webapp/src/dogma/common/components/editor/DeleteFileModal.tsx

```diff
@@ -0,0 +1,126 @@
+import {
+  Button,
+  FormControl,
+  FormErrorMessage,
+  Input,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Textarea,
+} from '@chakra-ui/react';
+import { SerializedError } from '@reduxjs/toolkit';
+import { FetchBaseQueryError } from '@reduxjs/toolkit/dist/query';
+import { usePushFileChangesMutation } from 'dogma/features/api/apiSlice';
+import { createMessage } from 'dogma/features/message/messageSlice';
+import ErrorHandler from 'dogma/features/services/ErrorHandler';
+import { useAppDispatch } from 'dogma/store';
+import Router from 'next/router';
+import { useForm } from 'react-hook-form';
+
+type FormData = {
+  summary: string;
+  detail: string;
+};
+
+export const DeleteFileModal = ({
```

Comment:
Side note: 
I saw 404 not found error message when I moved back after deleting a file.
<img width="1246" alt="image" src="https://user-images.githubusercontent.com/1866157/217471445-71c6666c-e7ad-4c5c-ba7d-fc73544b3ae3.png">
Should we render a simple error page than the raw JSON error message in future work?

Reply:
Agree. Perhaps integrating them with https://github.com/line/centraldogma/blob/new-webapp/webapp/src/dogma/features/services/ErrorHandler.ts.